iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpWithPasswordViewModel.swift

```diff
@@ -37,8 +37,8 @@ public final class SignUpWithPasswordViewModel: ObservableObject {
 		//TODO: API key to be handled in shared layer. remove the apikey parameter once implemented
 		// To run the project for now, add the api key and run
 		isLoading = true
-		firebase.signUp(apiKey: "AIzaSyCXmrUtOgzc4kj8aimSkmjOcCV9PR438-o", email: email, password: password, returnSecureToken: true, completionHandler: { result, error in
-			if (result?.idToken != nil) {
+		firebase.signUp(apiKey: "AIzaSyCXmrUtOgzc4kj8aimSkmjOcCV9PR438-o", email: email, password: password, returnSecureToken: true, completionHandler: { result, _ in
```

Comment:
Would be able to remove this key and share on DM?

Reply:
Yes totally, I removed it from my other PR but forgot about it here 😬